examples/db/redis-instance.yml

```diff
@@ -71,6 +71,14 @@ Parameters:
     MinLength: '8'
     MaxLength: '32'
     NoEcho: true
+  EngineVersion:
+    Type: String
+    Label:
+      en: Engine Version
+      zh-cn: 数据库版本
+    AllowedValues:
```

Comment:
EngineVersion already supports parameter query, AllowedValues ​​do not need to fill in